src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -0,0 +1,827 @@
+    confirmResolutionChkFormRef: FormIF,
+    confirmResolutionChkRef: FormIF
+  }
+
+  private MAX_RESOLUTION_TEXT_LENGTH: number = 1000
+  private resolutionText: string = ''
+  private hasResolutionConfirmed = false
+  private resolutionConfirmed = false
+  private helpToggle = false
+  private PLACEHOLDER_LEGAL_NAME = 'legal_name'
+  private PLACEHOLDER_LEGAL_NAME_INLINE = 'legal_name_inline'
+  private signingPerson: PersonIF = null
+
+  // Date properties
+  private resolutionDateText: string = ''
+  private signatureDateText: string = ''
+  private foundingDate: Date = null
+
+  @Getter getShowErrors!: boolean
+  @Getter isTypeCoop!: boolean
+  @Getter getNameRequestDetails!: NameRequestDetailsIF
+  @Getter getCreateResolutionResource!: CreateResolutionResourceIF
+  @Getter getCreateResolutionStep!: CreateResolutionIF
+  @Getter getValidateSteps!: boolean
+  @Getter getBusinessLegalName!: string
+  @Getter getBusinessFoundingDate!: string
+
+  @Action setResolution!: ActionBindingIF
+  @Action setResolutionStepValidationDetail!: ActionBindingIF
+
+  // Enum for template
+  readonly RouteNames = RouteNames
+  readonly ItemTypes = ItemTypes
+  readonly CorpTypeCd = CorpTypeCd
+  readonly BulletListTypes = BulletListTypes
+  readonly Rules = Rules
+
+  private get documentURL (): string {
+    const docUrl = sessionStorage.getItem('BASE_URL') +
+      this.getCreateResolutionResource.sampleFormSection.downloadDocPath
+    return docUrl
+  }
+
+  private previewImageSource (): string {
+    // Note: the image file path did not resolve correctly when using the require function directly.  In order
+    // to get the image path resolving correctly, needed to get the image context first and use that to build
+    // the final image file path
+    const images = require.context('@/assets/images/', false, /\.png$/)
+    return images('./' + this.getCreateResolutionResource.sampleFormSection.previewImagePath)
+  }
+
+  private get confirmLabel (): string {
+    const confirmTextItems = this.getCreateResolutionResource.confirmSection.text
+    const result = confirmTextItems.map(item => {
+      if (item.type === ItemTypes.PLACEHOLDER && item.value === this.PLACEHOLDER_LEGAL_NAME) {
+        return `<strong>${this.getBusinessLegalName}</strong>`
+      }
+      return item.value
+    }).join(' ')
+    return result
+  }
+
+  /** Validations rules for resolution date field. */
+  get resolutionDateRules (): Array<Function> {
+    const expectedDateFormat = /^(19|20)\d\d[-.](0[1-9]|1[012])[-.](0[1-9]|[12][0-9]|3[01])$/
+    return [
+      (v: string) => !!v || 'Select date',
+      (v: string) => expectedDateFormat.test(v) || 'Date format should be YYYY-MM-DD',
+      (v: string) =>
+        this.validateIsBetweenDates(this.resolutionDateMin,
+          this.resolutionDateMax,
+          this.createUtcDateFromStr(v)) ||
+        `Date should be between ${this.resolutionDateMinStr} (incorporation date) and ${this.resolutionDateMaxStr}`
+    ]
+  }
```

Comment:
I like it! 
We've recently introduced some `rule` files, would it make sense to relocate this rule set to be reusable from that location, or are they too specific you think?

Reply:
yeah, great idea.  The name ones were really handy.  I think some of these can go into a common file for sure.  Will look into it.

Reply:
Also, if these are to stay in this component, should they be `private` as well?

Reply:
yes, these should stay `private`.  good call.